forms.py

```diff
@@ -0,0 +1,18 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField, SelectField
+from wtforms.validators import DataRequired, Email, EqualTo, Length
+
+
+class RegistrationForm(FlaskForm):
+    firstname = StringField('Имя', validators=[DataRequired()])
+    lastname = StringField('Фамилия', validators=[DataRequired()])
+    user_login = StringField('Логин', validators=[DataRequired()])
+    gender = SelectField('Пол', choices=[('male', 'Мужчина'), ('female', 'Женщина')])
+    email = StringField('Почта', validators=[DataRequired(), Email()])
```

Comment:
А чтобы `EmailField` не использовать?
```suggestion
    email = EmailField('Почта', validators=[DataRequired(), Email()])
```